internal/tiltfile2/docker_compose.go

```diff
@@ -23,17 +24,21 @@ func (s *tiltfileState) dockerCompose(thread *skylark.Thread, fn *skylark.Builti
 	if err != nil {
 		return nil, err
 	}
+	absConfigPath, err := filepath.Abs(configPath)
```

Comment:
maybe ospath.RealAbs, like we did elsewhere?

Reply:
yes, thank you!